modules/lang-painless/src/test/java/org/elasticsearch/painless/PainlessDocGenerator.java

```diff
@@ -87,11 +88,11 @@ public static void main(String[] args) throws IOException {
                 Path typePath = apiRootPath.resolve(struct.name + ".asciidoc");
                 logger.info("Writing [{}.asciidoc]", struct.name);
                 try (PrintStream typeStream = new PrintStream(
-                        Files.newOutputStream(typePath, StandardOpenOption.CREATE_NEW, StandardOpenOption.WRITE),
-                        false, StandardCharsets.UTF_8.name())) {
+                    Files.newOutputStream(typePath, StandardOpenOption.CREATE_NEW, StandardOpenOption.WRITE),
```

Comment:
I think the indentation was better before, indicating these lines are a continuation of the try with resources.

Reply:
Completely agree.  Intellij is misbehaving.